lib/src/interface/local_platform.dart

```diff
@@ -48,5 +48,5 @@ class LocalPlatform extends Platform {
   String get version => io.Platform.version;
 
   @override
-  bool get ansiSupported => io.Platform.ansiSupported;
+  bool get ansiSupported => io.stdin.supportsAnsiEscapes;
```

Comment:
I wouldn't be too worried about that and just bump the major version number to indicate that we made incompatible API changes?